dist/s_executable

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Nitpick: `#!/usr/bin/env bash` (This calls `env`, which will locate `bash` itself - there might not be a `bash` in `/bin`, or they might be using a different, bash-compatible shell.)

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/10063/commits/4bf8930851b6ae2db75e2371cea38203d705710c